js/scripts/pre-push

```diff
@@ -0,0 +1,14 @@
+#!/bin/bash
+
+branch=$(git rev-parse --abbrev-ref HEAD)
+
+# Check that the branch starts with `ui` or `js`
+# and that `npm` is installed
+if [[ "$branch" =~ ^(ui|js) ]] && npm --version 1>/dev/null 2>/dev/null;
```

Comment:
Okay, agreed. So what about this :

- On `postinstall`, copy the git hook pre-push script to the right folder (like that everyone using npm will have it)
- The script itself only checks if `npm` is installed, as well as `eslint` (globally or in the `node_modules/bin` directory)